src/server/enterprise/cmds/cmds.go

```diff
@@ -192,6 +213,51 @@ func GetStateCmd() *cobra.Command {
 	return cmdutil.CreateAlias(getState, "enterprise get-state")
 }
 
+func SyncContextsCmd() *cobra.Command {
+	syncContexts := &cobra.Command{
+		Short: "Pull all available Pachyderm Cluster contexts into your pachctl config",
+		Long:  "Pull all available Pachyderm Cluster contexts into your pachctl config",
+		Run: cmdutil.Run(func(args []string) error {
+			cfg, err := config.Read(false, false)
+			if err != nil {
+				return err
+			}
+
+			ec, err := client.NewEnterpriseClientOnUserMachine("user")
+			if err != nil {
+				return errors.Wrapf(err, "could not connect")
+			}
+			defer ec.Close()
+
+			resp, err := ec.License.ListUserClusters(ec.Ctx(), &license.ListUserClustersRequest{})
+			if err != nil {
+				return err
+			}
+
+			// update the pach_address of all existing contexts, and add the rest as well.
+			for _, cluster := range resp.Clusters {
+				if context, ok := cfg.V2.Contexts[cluster.Id]; ok {
+					context.ClusterDeploymentID = cluster.ClusterDeploymentId
+					context.PachdAddress = cluster.Address
+				} else {
+					cfg.V2.Contexts[cluster.Id] = &config.Context{
+						ClusterDeploymentID: cluster.ClusterDeploymentId,
+						PachdAddress:        cluster.Address,
+						Source:              config.ContextSource_IMPORTED,
+					}
+				}
+			}
+
+			err = cfg.Write()
+			if err != nil {
+				return err
+			}
+			return nil
+		}),
+	}
+	return cmdutil.CreateAlias(syncContexts, "enterprise sync-contexts")
```

Comment:
@actgardner what do you think of this command name? Is there anyone else we should consult with?

Reply:
I think that's fine, it fits the convention of how we've been naming enterprise/auth/license/identity CLI commands.